doc/Makefile

```diff
@@ -7,7 +7,7 @@ aspect-files      = $(shell echo ../include/aspect/*.h \
                             ../include/aspect/*/*/*.h   \
                             modules/*.h)
 aspect-manual-pix = $(shell find manual/cookbooks/ -name \*.png 2>/dev/null)
-change-files = $(shell ls modules/changes)
+change-files = $(shell find modules/changes/ -type f -not -name '.*')
```

Comment:
I don't think I understand what this does. It selects all files that do not have the form `.*`? What files are left in that case?

Reply:
I added a comment to explain